algorithms/templates/shortest_ss_any_wt.gtmp

```diff
@@ -1,73 +1,87 @@
 *ATTR*CREATE QUERY shortest_ss_any_wt*EXT* (VERTEX source) FOR GRAPH *graph* {
 *ACCM*CREATE QUERY shortest_ss_any_wt*EXT* (VERTEX source, BOOL display) FOR GRAPH *graph* {
 *FILE*CREATE QUERY shortest_ss_any_wt*EXT* (VERTEX source, STRING filepath) FOR GRAPH *graph* {
-/* The Bellman-Ford algorithm for single-Source Shortest Path with edge weights,
- possibly negative.
- If any loop in the graph has a net negative weight, the algorithm will exit.
+/* The Bellman-Ford algorithm for single-Source Shortest Path 
+   on directed/undirected graph with real value weight.
+   It will always detect negative cycle if an undirected graph is used. 
```

Comment:
Maybe we can make this a little bit more clear:

once a negative cycle detected, an exception will be raised